OtripleS.Portal.Web/Brokers/Apis/IApiBroker.cs

```diff
@@ -0,0 +1,12 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE AS LONG AS SOFTWARE FUNDS ARE DONATED TO THE POOR
+// ---------------------------------------------------------------
+
+namespace OtripleS.Portal.Web.Brokers.Apis
+{
+    public partial interface IApiBroker
+    {
+
```

Comment:
We can get rid of this space

Reply:
I was able to do the build without no problem

![image](https://user-images.githubusercontent.com/53588704/140184448-5defeb7f-111c-4b44-8217-48724320989f.png)

Reply:
Interesting. I was referring to these where the namespace still needs 'Apis'

![image](https://user-images.githubusercontent.com/89320816/140251145-da420e2d-d22d-4a30-a9a8-87661a406fbe.png)

Reply:
Thank you for the clarification. I made the changes